### PR TITLE
D3ASIM-1170: Improved AreaSerialiser.

### PR DESCRIPTION
**Pull request by hannesdiedrich**
This is not the fix of the bug (its done in d3a-web).
This is just an add on, that enables us to use area_serializer.area_to_string() to use for debugging.

### CODE REVIEW

File: src/d3a/models/strategy/storage.py [hunk 1]
```diff
@@ -60,6 +60,10 @@ def __init__(self, risk: int = GeneralSettings.DEFAULT_RISK,
                  cap_price_strategy: bool=False,
                  min_allowed_soc=None):
 
+        self.initial_capacity_kWh = initial_capacity_kWh
```
Comment (spyrostz) [inline src/d3a/models/strategy/storage.py]: Why are these needed here? I would suggest to leave them only in the state class. The disadvantage that the duplication introduces is that any change that happens to the variables of the StorageStrategy class will be independent from any change that happens to their clones in StorageState class.

Comment (hannesdiedrich) [reply src/d3a/models/strategy/storage.py]: Makes sense. The lack of these self members threw an error in the areaSerialiser so i added them without thinking. Will remove them here and add a catch in the serializer.
